butterknife-compiler/src/main/java/butterknife/compiler/BindingSet.java

```diff
@@ -76,6 +85,7 @@ private BindingSet(TypeName targetTypeName, ClassName bindingClassName, boolean
     this.isView = isView;
     this.isActivity = isActivity;
     this.isDialog = isDialog;
+    this.useAndroidX = useAndroidX;
```

Comment:
This seems like something we should inject into `brewJava` since it's not a property of the things being bound but rather an implementation detail of _how_ the bindings are generated, just like the SDK level and whether or not the variant is debugging.

Reply:
a479d06

Reply:
how can I add support for AndroidX